### PR TITLE
upgrade pillow version to fix issues running on docker

### PR DESCRIPTION
**Pull request by thejerrytan**
After running 'roslaunch launch/styx.launch' from the Docker image while the simulator is running and the connection is established, then once I click on the camera check box, I get thse errors in ros (the errors are repetitive as long as there's a connection):

```
message handler error
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/engineio/server.py", line 405, in _trigger_event
    return self.handlers[event](*args)
  File "/usr/local/lib/python2.7/dist-packages/socketio/server.py", line 509, in _handle_eio_message
    self._handle_event(sid, pkt.namespace, pkt.id, pkt.data)
  File "/usr/local/lib/python2.7/dist-packages/socketio/server.py", line 448, in _handle_event
    self._handle_event_internal(self, sid, data, namespace, id)
  File "/usr/local/lib/python2.7/dist-packages/socketio/server.py", line 451, in _handle_event_internal
    r = server._trigger_event(data[0], namespace, sid, *data[1:])
  File "/usr/local/lib/python2.7/dist-packages/socketio/server.py", line 480, in _trigger_event
    return self.handlers[namespace][event](*args)
  File "/capstone/ros/src/styx/server.py", line 60, in image
    bridge.publish_camera(data)
  File "/capstone/ros/src/styx/bridge.py", line 182, in publish_camera
    image_message = self.bridge.cv2_to_imgmsg(image_array, encoding="rgb8")
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/cv_bridge/core.py", line 248, in cv2_to_imgmsg
    img_msg.height = cvim.shape[0]
IndexError: tuple index out of range
```

Fix is to upgrade Pillow version. Mentioned here:
https://github.com/udacity/CarND-Capstone/issues/147

### CODE REVIEW

**Comment by ser94mor** [comment]
But since Pillow version is fixed in `requirements.txt` file, we cannot just change it to another version. Carla will still have the older version. 

See, it is not easy to update the environment on Carla: https://github.com/udacity/CarND-Capstone/issues/272.

I think, that by merging this PR we potentially produce big compatibility problems.

**Comment by thejerrytan** [comment]
anyone else getting this tuple index error when running in docker?

**Comment by thejerrytan** [comment]
actually i think its fine, the temp getaround would be to:

pip uninstall Pillow

and 

pip install Pillow==4.3

**Comment by pkorivi** [comment]
I think we cant change environment on Carla, it's only for local work. Let's not merge the pull request but if someone needs a new version can try to install one.

**Comment by thejerrytan** [comment]
i have added a workaround:

in dockerfile
`ENTRYPOINT ["bash", "../entrypoint.sh"]`


entrypoint.sh
```bash
echo "source /opt/ros/kinetic/setup.bash" >> ~/.bashrc
source ~/.bashrc
pip uninstall --yes Pillow
yes | pip install Pillow==4.3
bash
```

File: Dockerfile
Comment (ser94mor) [inline Dockerfile]: I think CMD would be more suitable here. You can override it by simply providing different command after the image name in the *docker run* command. To override ENTRYPOINT you have to use *--entrypoint* which is a bit more inconvenient. 

Also, do not forget to include the newline at the end of both files. GitHub shows the symbols indicating this.

**Review by ser94mor — APPROVED** [review]

**Comment by ser94mor** [comment]
@thejerrytan, are you planning to merge this PR? If yes, please, take actions to get it approved. If not, please, close this PR.

**Comment by thejerrytan** [comment]
Hang on let me switch to CMD and we can merge it in

**Pull request merged**
